endroid/prediction-io/2.3/manifest.json

```diff
@@ -0,0 +1,12 @@
+{
+    "bundles": {
+        "Endroid\\PredictionIO\\Bundle\\PredictionIOBundle\\EndroidPredictionIOBundle": ["all"],
+        "Endroid\\PredictionIO\\Bundle\\PredictionIODemoBundle\\EndroidPredictionIODemoBundle": ["all"]
```

Comment:
the demo bundle should not be registered when installing the package in a project. It does not make sense to have it in actual projects.

Reply:
btw, I would even suggest splitting the demo into a totally separate package, so that it does not even need to be installed.